src/matter/cluster/server/LevelControlServer.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * @license
+ * Copyright 2022 The node-matter Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { LevelControlCluster } from "../LevelControlCluster";
+import { ClusterServerHandlers } from "./ClusterServer";
+import { UseOptionalAttributes } from "./ClusterServer";
+
+// TODO: Create temporary options based on mask and override. How to expose to user of the library?
+
+export const LevelControlClusterHandler: () => ClusterServerHandlers<typeof LevelControlCluster> = () => ({
+    moveToLevel: async ({ request:{level, transitionTime, optionsMask, optionsOverride}, attributes: { currentLevel } }) => { 
```

Comment:
nit: no need to reference parameters that you are not using
moveToLevel: async ({ request: { level }, attributes: { currentLevel } }) => {